src/Form/FormControl.jsx

```diff
@@ -122,6 +125,8 @@ FormControl.propTypes = {
   isInvalid: PropTypes.bool,
   /** Only for `as="textarea"`. Specifies whether the input can be resized according to the height of content. */
   autoResize: PropTypes.bool,
+  /** Specifies what format to use for the input mask. */
+  hasInputMask: PropTypes.string,
```

Comment:
Usually, if a variable's name starts with `has` it indicates that it is of boolean type, in your case it's a string, so it would be better to name it just `inputMask`.

Reply:
Corrected, thanks